api/src/server/gql/internal.rs

```diff
@@ -19,11 +19,9 @@ const ALL_NODES_GET_BY_ID: &[&dyn Fn(
 
 /// A trait to identify any struct that is a GQL node.
 pub trait NodeType: Sized + Into<Node> {
-    /// The DB model type associated with this node type.
-    type Model;
-
-    /// Convert the underlying model value to a value of this type.
-    fn from_model(model: Self::Model) -> Self;
+    /// The DB model type associated with this node type. Requires `Into<Self>`
+    /// so that we can convert models into this type.
+    type Model: Into<Self>;
```

Comment:
I removed this function in favor of adding a bound on `Into<Self>`. This provides the same functionality, but now we get the language's built into `Into`/`From` ergonomics. I tried to do this originally but couldn't figure it out cause I was being dumb.